liblangutil/Token.cpp

```diff
@@ -126,6 +126,8 @@ int precedence(Token tok)
 }
 #undef T
 
+namespace
+{
```

Comment:
Ok - that's nice to ask for - but which warning is that :-)? Didn't see one that would fit complaining about this actually, when I went through the list before :-D.

Reply:
`-Wmissing-declarations`

Reply:
I did - just didn't read that one to the end :-).